### DISCUSSION TITLE
Notes on Contributing: Issues and Pull Requests

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/caMicroscope#710] [Announcements]
Hi! I thought I'd quickly share a bit about how caMicroscope usually uses issues and Pull Requests (PRs)!
We don't usually assign issues without a very compelling reason (e.g. update the demo server may be assigned to me, since most people don't have access). So this means that issues are pretty much open to a good PR.

Of course, this may mean that an issue may have multiple PRs addressing it. Sometimes, we can merge multiple of these with little/no problem. Sometimes we may ask a PR author to fix a merge conflict before adding it.

If you want to work on a PR in public but aren't finished, you can mark it as WIP either by setting it as a draft in github, or if you can't figure out how to do that, just mark it as WIP somehow clearly. Be sure to unmark it as wip/draft when finished!

**Comment by gbengaoluwadahunsi** [comment]
noted  @birm

**Comment by We13b-MD** [comment]
Ok @birm

**Comment by barchakuz** [comment]
acknowledged @birm

**Comment by LeonardAzah** [comment]
clear @birm

**Comment by nwanduka** [comment]
Noted. Thanks for the information

**Comment by Kingjhenny** [comment]
Noted @birm

**Comment by Arnav2824** [comment]
Does that mean that I will have to generate my own issue and you will go through it and approve it first and then I will be able to generate a pull request on it or can I generate a pull request on any of the issues that have already been raised? 
Sorry, I am just a bit unclear about the things and want to make sure I am going in the right direction.

**Reply by birm** [reply]
You shouldn't have to generate your own issue, just make sure we can understand what a PR is trying to do.
We're a little behind on PR review right now, however. Hoping to work on catching up soon.

**Comment by Lahitan** [comment]
noted @birm

**Comment by Rishi2003Das** [comment]
@birm 
In one of your ML Classification Models, I want to add an accuracy or prediction check at the end. I have modified the code in my branch in the forked repository. Now can you please guide me on a further approach? I am new to the OS community but enthusiastic to contribute. Please help sir at your convenience.
